test/server/lib/db/query.js

```diff
@@ -5,44 +5,44 @@ const rewire = require('rewire');
-    describe('searchTotals function', () => {
-        // eslint-disable-next-line no-underscore-dangle
-        const totalCountSQL = query.__get__('totalCount');
-        // eslint-disable-next-line no-underscore-dangle
-        const forTodaySQL = query.__get__('forToday');
-        let fakeQuery;
-        let stub;
-        before(() => {
-            stub = sinon.stub();
-            stub.returns(Promise.resolve());
-            fakeQuery = proxyquire('../../../../src/lib/db/query', {
-                './postgres': { manyOrNone: stub }
-            });
-        });
-        describe('when `true` is provided', () => {
-            it('should return a promise', () =>
-                expect(fakeQuery.searchTotals(true))
-                    .to.be.an.instanceOf(Promise)
-                    .that.is.fulfilled
-            );
-            it('should pass SQL to the database library', () =>
-                expect(stub).to.have.been.calledOnce
-                    .and.to.have.been.calledWithExactly(totalCountSQL)
-            );
-        });
-        describe('when `false` is provided', () => {
-            before(() => {
-                stub.resetHistory();
-            });
-            it('should return a promise', () =>
-                expect(fakeQuery.searchTotals(false))
-                    .to.be.an.instanceOf(Promise)
-                    .that.is.fulfilled
-            );
-            it('should pass SQL to the database library with the "today" where clause', () =>
-                expect(stub).to.have.been.calledOnce
-                    .and.to.have.been.calledWithExactly(totalCountSQL + forTodaySQL)
-            );
-         });
-    });
+	describe('searchTotals function', () => {
+		// eslint-disable-next-line no-underscore-dangle
+		const totalCountSQL = query.__get__('totalCount');
+		// eslint-disable-next-line no-underscore-dangle
+		const forTodaySQL = query.__get__('forToday');
+		let fakeQuery;
+		let stub;
+		before(() => {
+			stub = sinon.stub();
+			stub.returns(Promise.resolve());
+			fakeQuery = proxyquire('../../../../src/lib/db/query', {
+				'./postgres': { one: stub }
+			});
+		});
+		describe('when `true` is provided', () => {
+			it('should return a promise', () =>
+					expect(fakeQuery.searchTotals(true))
+							.to.be.an.instanceOf(Promise)
+							.that.is.fulfilled
+			);
+			it('should pass SQL to the database library', () =>
+					expect(stub).to.have.been.calledOnce
+							.and.to.have.been.calledWith(totalCountSQL)
+			);
+		});
+		describe('when `false` is provided', () => {
+			before(() => {
+				stub.resetHistory();
+			});
+			it('should return a promise', () =>
+					expect(fakeQuery.searchTotals(false))
+							.to.be.an.instanceOf(Promise)
+							.that.is.fulfilled
+			);
+			it('should pass SQL to the database library with the "today" where clause', () =>
+					expect(stub).to.have.been.calledOnce
+							.and.to.have.been.calledWith(totalCountSQL + forTodaySQL)
+			);
+		});
+	});
```

Comment:
I think these indentation changes are wrong.

I'm guessing your editor did it.

We tend to use 2 spaces rather than tabs for indentation.

In this case it looks like the original was set to 4 spaces.

Any chance we could fix it prior to merging?

Reply:
Fixed

Reply:
Are you sure? - It's not showing as fixed for me.